styleguide/_themes/derek/scss/globalElements/mainnav.scss

```diff
@@ -447,4 +469,18 @@
       background: transparent;
     }
   }
+
+  .navbar-nav {
+
+    .navbar-topItem {
+      font-size: 1.45rem;
+
+      .navbar-dropDownLink,
+      .navbar-dropDownLabel,
+      .navbar-tertiary-dropDownLink {
+        border-bottom: 1px solid $gray-darker;
+        font-size: 1.45rem;
+      }
+    }
+  }
```

Comment:
Can confirm the necessity of this three-level nesting?  Is this to override bootstrap specificity?